runtime/doc/builtin.txt

```diff
@@ -796,6 +796,8 @@ complete_info([{what}])                                        *complete_info()*
 				dictionary containing the entries "word",
 				"abbr", "menu", "kind", "info" and "user_data".
 				See |complete-items|.
+		   pum_items	Indices of completion matches that are shown
```

Comment:
There is already an `items` field which is a "list of completion matches". The field `pum_items` then suggests that the value has the same structure as the `items`, which it does not (and rightfully so).

As the new field is more aligned with `selected` field, maybe `shown` is a better name? Using suggested `shown_idx` is also good, just does not fit as well here.

Reply:
> There is already an `items` field which is a "list of completion matches". The field `pum_items` then suggests that the value has the same structure as the `items`, which it does not (and rightfully so).
> 
> As the new field is more aligned with `selected` field, maybe `shown` is a better name? Using suggested `shown_idx` is also good, just does not fit as well here.

How about `pum_shown`?

Reply:
> How about `pum_shown`?

Yep, also seems fine.

Reply:
oops, accidentally deleted my comment

Reply:
`shown` is probably a better name since `selected` is also related to PUM but doesn't have a `pum_` prefix